core/src/test/scala/io/udash/properties/PropertyTest.scala

```diff
@@ -560,6 +560,41 @@ class PropertyTest extends UdashCoreTest {
       sqc.get should be(Seq(10, 5))
     }
 
+    "trigger once for multiply combined properties" in {
+      val calls = mutable.Buffer.empty[(Int, Int, Int)]
+
+      val p0 = Property.blank[Int]
+      val p1 = Property.blank[Int]
+      val p2 = Property.blank[Int]
+
+      p0.combine(p1)((_, _))
+        .combine(p2)((_, _))
+        .listen { case ((i1, i2), i3) =>
+          calls.append((i1, i2, i3))
+        }
+
+      CallbackSequencer().sequence {
+        p0.set(0)
```

Comment:
`p0` default value is `0` (resulting from corresponding `Blank`), hence this line has no effect which is confusing a little bit when digging into the `CallbackSequencer` details - please turn the `force` flag on or change the value